### ISSUE TITLE
tried nflmodel update and its telling me that its not an option

### ISSUE THREAD

**Issue by copypasteearth** [morelandjs/nfl-model#1]
hello,

Im trying to use your project to predict nfl games and its telling me that nflmodel update is not an option

**Comment by Dangaroo** [comment]
Same problem here, the forecast argument does not exist either.

**Comment by matthoffner** [comment]
Yeah those look deprecated with the changes to the underlying data source.
